src/nr_openai_observability/build_events.py

```diff
@@ -0,0 +1,150 @@
+import uuid
+from datetime import datetime
+
+import openai
+
+
+def _build_messages_events(messages, completion_id, model):
+    message_id = str(uuid.uuid4())
```

Comment:
Wanted to check my understanding here. Should we be generating one UUID for each message here vs a UUID for all messages in the "completion"?

Reply:
We have a completionID, which is a UUID generated for each completion.

The question of whether or not we should be generating a UUID for each message event, or can we relay on completionID+sequence is debatable, and can be changed.